### DISCUSSION TITLE
How can I increase disk space

### DISCUSSION THREAD

**Discussion by shao60533** [Whisky-App/Whisky#259] [Q&A] [answered]
My c drive space has 40G,but Diablo 4 need 90G. How can I increase disk space？

**Comment by ferreira-ff** [comment]
Hello. Create a bottle, then move it to an external hd or ssd. After that, place a “simlink” on the Whisky’s Library folder.

Kind regards

**Reply by shao60533** [reply]
how can i place a simlink. I will really appreciate if you write step by step.

**Comment by ferreira-ff** [comment]
Part **1 of 2**

**Main Wisky application files & folders:**
_Application: /Applications/Whisky.app (in my case: ~/Applications/Whisky.app)_
_Engines: ~/Library/Application\ Support/com.isaacmarovitz.Whisky_
_Bottles: ~/Library/Containers/com.isaacmarovitz.Whisky_
_Configuration: /Library/Preferences/com.isaacmarovitz.Whisky.plist_

**_If needed, press (Command + Shift + .) to show hidden files and folders; press the same combination again, to hide them_**

**Part 2 of 2**

**_I am going to use Steam as an example. Let's say you want to have Steam and all the games, on a bottle, on an external SSD_**

1. Open Whisky and create the Steam bottle. Install the Windows version of Steam and configure it, to your liking
The Whisky created bottle, will be **something like this**:
(~/Library/Containers/com.isaacmarovitz.Whisky/Bottles/**98E17132-BD5D-4388-944B-AD22F5C2E455**)

_The part you need, to start having fun, with your game..._

2. Copy the bottle **98E17132-BD5D-4388-944B-AD22F5C2E455** to the external SSD
_Erase the bottle **98E17132-BD5D-4388-944B-AD22F5C2E455** on the Mac internal SSD_

3. Select the bottle on the external SSD, right click on it and chose "**create alias**"
_A new link like this (**98E17132-BD5D-4388-944B-AD22F5C2E455 alias**) will be created_
Erase the **alias** part of the name; it only should be **98E17132-BD5D-4388-944B-AD22F5C2E455** as the name

4. Copy the created alias (**98E17132-BD5D-4388-944B-AD22F5C2E455**)
to ~/Library/Containers/com.isaacmarovitz.Whisky/Bottles/
Erase the created alias (98E17132-BD5D-4388-944B-AD22F5C2E455) on the external SSD

_Now; every time the Whisky.app will search for the Steam bottle, it will follow the route to the external SSD_

~/Library/Containers/com.isaacmarovitz.Whisky/Bottles/**(SimLink)98E17132-BD5D-4388-944B-AD22F5C2E455** > **(External SSD Folder)98E17132-BD5D-4388-944B-AD22F5C2E455**

It looks a little confusing, but actually it is very simple. I think you will find "Symbolic Links" very useful; you can use them, for a lot of things. I hope I did not forget anything. Kind regards.

**Comment by olmi02** [comment]
I have the same issue.

Why do I need an external drive if I have enough free space on my internal disk?

I have more than 100GB free, but the BattleNet-Installer only shows free space of 36,7GB. I can't install Diablo4.

Thanks for your help!

**Reply by ferreira-ff** [reply]
Hello again. I think that is not Whisky's fault _(Wine 7.7.0)_. Anyway; Whisky is good for "bottles"...

If you are interested only on the **Battle.net + Diablo 4** game, you could try:

[WineskinServer](https://github.com/Gcenx/WineskinServer)
Wineskin-2.9.1.9 & WineCX22.1.1-8

_(1)_ Create the wrapper, _(2)_ "ditto command" the GPTK 0.2 and then, _(3)_ rebuild the wrapper.
That is what I have done for my daughters, so they can play **Overwatch 2**.

Good information here: https://github.com/Gcenx/WineskinServer/issues/420
**Using D3DMetal with Wineskin Engine(s) #420**

Kind regards to all

VERY IMPORTANT: WHEN INSTALLING the  Battle.net Launcher, ON THE CONFIGURATION, **DISABLE HARDWARE ACCELERATION**; DO NOT CHOSE APPLY AND RELAUNCH. SIMPLY CLOSE THE LAUNCHER / WRAPPER, WAIT AND THEN RESTART THE WRAPPER.

**Reply by shao60533** [reply]
hello friend, I didn't get step 2 GPTK. I create the wrapper and it shows whole disk space is only 13GB.
Thanks for your help!

**Reply by ferreira-ff** [reply]
Hello and good day. I think you are referring to Wineskin. Mr. Gcenx, the creator of WineskinServer, explains it very easily here:
**Using D3DMetal with Wineskin Engine(s) #420** https://github.com/Gcenx/WineskinServer/issues/420

Kind regards

**Comment by IsaacMarovitz** [comment, marked as answer]
Wine uses macOS APIs to detect if you have free space available. Sometimes, macOS will underreport this figure. You can try restarting, clearing your trash, among other common cleanup tasks, to free up space.

**Reply by nofesfunnyface** [reply]
I know this is a YEAR later, but I'm having the same problem with GTA 5.
How long will I have to wait until it is not 'underreporting' the avaliable space.
It says I only have "50GB" (GTA 5 is 112gb) but in reality I have 150GB (maybe even more) space avaliable.
When I go into settings it says I have 150GB but Whisky is full-on underreporting it.

Just want to know how long I'm going to have to wait for the avaliable space to update.

**Comment by E-VANCE** [comment]
For me the issue was tied to Mac OS (Sonoma) allocating disk space behind the scenes for TimeMachine – the actual free disk space reported by Disk Utility greatly differed from what was stated in the finder...

Run `tmutil listlocalsnapshots /` to list them and use `tmutil deletelocalsnapshots <date and time>` to get rid of those.

**Reply by tomearly** [reply]
This did it for me... ended up with 219gb instead of 30gb
